config/300-serving.yaml

```diff
@@ -739,6 +739,47 @@ spec:
                               type: array
                           type: object
                       type: object
+                    resources:
+                      description: If specified, the container's resources.
+                      items:
+                        description: The pod this Resource is used to specify the requests and limits for
+                          a certain container based on the name.
+                        properties:
+                          container:
+                            description: The name of the container
+                            type: string
+                          limits:
+                            properties:
+                              cpu:
+                                pattern: ^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
+                                type: string
+                              ephemeral-storage:
+                                pattern: ^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
+                                type: string
+                              memory:
+                                pattern: ^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
+                                type: string
+                              storage:
```

Comment:
Same with `ephemeral-storage`. It is not used.

Reply:
it is defined in spec.resources already as well.